packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -351,13 +352,18 @@ class DebuggerController extends DisposableController
     _librariesVisible.value = !_librariesVisible.value;
   }
 
-  IsolateRef isolateRef;
-  bool get isSystemIsolate => isolateRef?.isSystemIsolate ?? false;
+  IsolateRef get isolateRef {
+    return _isolateRef!;
+  }
+
+  IsolateRef? _isolateRef;
```

Comment:
if _isolateRef can actually be null, we should not be unconditionally null-asserting here. If it shouldn't be null, then we should try to make _isolateRef late so that it can be non-nullable